tests/kernel/wave/wave_e2e_test.py

```diff
@@ -994,6 +995,22 @@ def test_igemm_conv(n, h, w, c, hf, wf, nf, stride, mem_space, layout, request):
         )
     ]
 
+    hyperparams = {
+        N: n,
+        C: c,
+        W: w,
+        H: h,
+        NF: nf,
+        WF: wf,
+        HF: hf,
+        BLOCK_M: 64,
+        BLOCK_N: 128,
+        BLOCK_K: 32,
+        ELEMS_PER_THREAD: 4,
+        ADDRESS_SPACE: mem_space,
+    }
+    hyperparams.update(get_default_scheduling_params())

```

Comment:
can't we just integrate into the backend directly? since we may use this area to override default variables

Reply:
Would be good to add it here
https://github.com/iree-org/iree-turbine/blob/e3b6c878f3f0ee26660e6d4762879bff0c8394a3/iree/turbine/kernel/wave/scheduling/resources.py#L32-L40

So instead of `return np.array([int(subs_idxc(x)) for x in resources]) ` we can do something like:
```python
default_resources =  get_default_scheduling_params()
return np.array([int(subs_idxc(x)) if x in subs_map else default_resources[x] for x in resources]) 
```

and similar method for the delays as well. I think if we can do this way we would not need to update hyperparameter for every kernel and it makes overwriting default values cleaner. :)

Reply:
Those values are not really 'default' as they can be different for different GPUs, it's more like `get_default_device_scheduling_params()` so I think it's better to set them explicitly.